gtk4/src/auto/im_context.rs

```diff
@@ -183,6 +191,27 @@ impl<O: IsA<IMContext>> IMContextExt for O {
         }
     }
 
+    fn get_surrounding_with_selection(&self) -> Option<(glib::GString, i32, i32)> {
```

Comment:
Missing `Since`

Reply:
Fixed upstream https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3315, let's wait till the next patch release if there would be any

Reply:
Actually, as it's small enough of a change, I patched it here as well :)